test-data/common.h

```diff
@@ -0,0 +1,12 @@
+#include "vmlinux.h"
```

Comment:
Maybe in future we can create a header folder for all header files..but fine for now.

Reply:
Thank you for your review. 
How do you feel about use `void` instead of `struct sched_process_folk_t`?  

for example:
````c
SEC("tracepoint/sched/sched_process_fork")
int sched_process_fork(void *ctx) {
    return 0;
}
````

Reply:
Yeah should be fine for now since we are not evaluating taking any action from this function and used just for testing purpose.

Reply:
fix to use `void` instead of the struct.